### PR TITLE
Updated University of Texas at Arlington faculty list.

### PR DESCRIPTION
**Pull request by akib-zaman**
…ated the information of a few new faculty members.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
You have incorrectly removed the numeric suffixes (e.g., "0001") from a number of faculty. I am not sure why you did this; please leave them intact. I will the merge once those are restored.

File: csrankings.csv [hunk 1]
```diff
@@ -2631,11 +2643,15 @@ Daniel Jorge Viegas Gonçalves,Universidade de Lisboa,https://fenix.tecnico.ulis
...
 Daniel Kroening,University of Oxford,https://www.cs.ox.ac.uk/people/daniel.kroening,DHddutUAAAAJ
+Daniel Král,Masaryk University,https://www.muni.cz/en/people/44742,crv1hRwAAAAJ
+Daniel Král',Masaryk University,https://www.muni.cz/en/people/44742,crv1hRwAAAAJ
 Daniel Kröning,University of Oxford,https://www.cs.ox.ac.uk/people/daniel.kroening,DHddutUAAAAJ
 Daniel L. Chester,University of Delaware,https://www.eecis.udel.edu/~chester,f1rXXY8AAAAJ
 Daniel L. Pimentel-Alarcón,Georgia State University,https://danielpimentel.github.io/index.html,NOSCHOLARPAGE
 Daniel Leithinger,University of Colorado Boulder,https://www.colorado.edu/atlas/daniel-leithinger,OsJr4BMAAAAJ
 Daniel Leivant,Indiana University,https://www.cs.indiana.edu/~leivant,NOSCHOLARPAGE
+Daniel Lokshantov,University of California - Santa Barbara,http://www.cs.ucsb.edu/people/faculty/lokshtanov,vaQGFLsAAAAJ
+Daniel Lokshtanov,University of California - Santa Barbara,http://www.cs.ucsb.edu/people/faculty/lokshtanov,vaQGFLsAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:2654]: Please remove duplicate.

**Comment by emeryberger** [comment]
Please resolve conflicts and try again. Thanks.

**Comment by dajiangzhu** [comment]
Hi, we just submitted the file and wondering when you will update the DB next time? Is it possible to do it recently? Thanks!

**Comment by emeryberger** [comment]
Am I mistaken or are there a number of UTA people being deleted for some reason? (For example, Jean Gao)

**Comment by akib-zaman** [comment]
There were duplicate entries for Jean Gao so I have removed that. And also a few professors left the university so they have been removed.

________________________________

**Comment by emeryberger** [comment]
The Jean Gao entry in particular appears to be removed entirely.

**Comment by dajiangzhu** [comment]
Yes. There are two entries for Jean (see below) and according to DBLP (https://dblp.org/search?q=jean%20gao), “Jean X. Gao” is alias of “Jean Gao” in DBLP.
Jean X. Gao,University of Texas at Arlington,http://crystal.uta.edu/~gao,NOSCHOLARPAGE
Jean Gao,University of Texas at Arlington,http://crystal.uta.edu/~gao,NOSCHOLARPAGE
So, we remove one of them.

**Comment by emeryberger** [comment]
No. You must leave all aliases in place. From the instructions, which you checked:

> If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in dblp-aliases.csv, add it.

**Comment by dajiangzhu** [comment]
We just recovered the removed entry back. Let us know if there are other issues. Thanks so much!

**Pull request merged**